generate_test_dataset.py

```diff
@@ -15,16 +15,18 @@
                         help="maximum number of texts to store into database per method")
     parser.add_argument("--queries", action="store", type=str, required=False,
                         help="scrape articles for a given query, insert multiple values comma separated")
+    # humans can be either human-verified or human (unverified) as that depends on the url
     parser.add_argument("--methods", action="store", type=str, required=False, default="",
-                        help="what data should be part of the test data, available are human, gpt2, gpt3, gpt4, gemini, grover")
+                        help="what data should be part of the test data, available are gpt2, gpt3, gpt4, gemini, grover, and human-verified or human")
 
     args = parser.parse_args()
     methods = args.methods.split(",")
+    visited = []
```

Comment:
this could be a set, O(n) to check this list would ruin our amazing performance :p